### PR TITLE
Hello microverse work

### PR DESCRIPTION
**Pull request by DammyShittu**
This is the code for the first Microverse coding exercise on the dashboard.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
```
Comment (rloterh) [inline style.css]: - [ ] Some styelint linter checks are failing.

**Stylelint errors:**
image

To ensure a cleaner code kindly correct all styelint errors.

File: README.md [hunk 2]
```diff
@@ -1 +1,29 @@
-# Hello-Microverse
```
Comment (rloterh) [inline README.md]: - [ ] Your ReadMe does is not descriptive enough. Always remember that your ReadMe is the first place most visitors of your repository will initially look at, and therefore it must be descriptive/informative clear, and well structured. Please use [this template](https://github.com/microverseinc/readme-template) to ensure a well-structured ReadMe but remember to customize it to your project.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Dammy, 

Welcome to your first project😄, you got most right!👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there!👍 

## Required Changes ♻️
- [ ] _Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
__As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).__

**Review by Shaher-11 — COMMENTED** [review]
## _Status:_ Invalid:no_entry::no_entry::no_entry:
### Hello @shaher
This Code Review Request will be marked as invalid in your dashboard because the linter checks are not working as required for this project.
Please make the necessary changes and submit another code review request once you are ready! :smiley: :smiley:

**Review by Shaher-11 — APPROVED** [review]
#### Hello @DammyShittu,

# Congratulations! 🎉 
#### Your project is approved!  There is nothing else to say other than... it's time to merge it :muscle: :muscle::muscle:

### Optional Suggestions💡
_The following point is optional to implement however it is highly recommended!_:smile::smile:

- Add a screenshot to the readme file.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Cheers and Happy coding!👏👏👏

**Pull request merged**